data/test/sort_datasource_impl_test.dart

```diff
@@ -116,6 +116,15 @@ void sortFilesTest() {
           Sorter.fromOrderScreen(OrderScreen.sharedSpaceDocument));
       expect(result, [workGroupDocument1, workGroupDocument2, workGroupFolder1]);
     });
+
+    test('sortFiles should return success with list received shares has been sorted sender date', () async {
+      await _initDataSource();
+
+      final result = await _sortDataSourceImpl.sortFiles(
+          [receivedShare1, receivedShare2],
+          Sorter.fromOrderScreen(OrderScreen.receivedShares));
+      expect(result, [receivedShare1, receivedShare2]);
```

Comment:
You should use the input of another range so that you can make a clear comparison when doing sorting. For example: `[receivedShare2, receivedShare1]` => `expect(result, [receivedShare1, receivedShare2]);`